packages/graphile-build-pg/src/plugins/PgBasicsPlugin.js

```diff
@@ -160,17 +160,19 @@ const omitUnindexed = omit => (
       if (!entity._omitUnindexedReadWarningGiven) {
         // $FlowFixMe
         entity._omitUnindexedReadWarningGiven = true;
-        // eslint-disable-next-line no-console
-        console.log(
-          "%s",
-          `Disabled 'read' permission for ${describePgEntity(
-            entity
-          )} because it isn't indexed. For more information see https://graphile.org/postgraphile/best-practices/ To fix, perform\n\n  CREATE INDEX ON ${`"${
-            klass.namespaceName
-          }"."${klass.name}"`}("${entity.keyAttributes
-            .map(a => a.name)
-            .join('", "')}");`
-        );
+        if (!hideIndexWarnings) {
```

Comment:
Actually, on second thoughts, can you merge this `if` clause with the one above; something like:

```js
const shouldOutputWarning = !hideIndexWarnings && !entity._omitUnindexedReadWarningGiven;
if (!shouldOutputWarning) {
  entity._omitUni...
```

It'll reduce the unnecessary work further (namely defining that new property on the entity) and also reduce the indentation depth.

Reply:
Sure. I left that out of the if block on purpose because I thought `entity._omitUnindexedReadWarningGiven = true;` needs to run either way as this might be checked at some other place. Didn't wanna mess with that. :-)